src/App.tsx

```diff
@@ -8,53 +9,115 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { getUsersFromServer } from './api/users';
+import { Post } from './types/Post';
+import { getPostsFromServer } from './api/posts';
 
-export const App = () => (
-  <main className="section">
-    <div className="container">
-      <div className="tile is-ancestor">
-        <div className="tile is-parent">
-          <div className="tile is-child box is-success">
-            <div className="block">
-              <UserSelector />
-            </div>
+export const App = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [selectedUserId, setSelectedUserId] = useState<number | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [notificationMessage, setNotificationMessage] = useState('');
+  const [activePostId, setActivePostId] = useState<number | null>(null)
```

Comment:
It seems you forgot to add a semicolon at the end of this line. While this is not a critical issue in JavaScript, it's a good practice to include semicolons to avoid potential issues.